lib/devicetrust/testenv/fake_device_service.go

```diff
@@ -90,13 +92,22 @@ func (s *fakeDeviceService) EnrollDevice(stream devicepb.DeviceTrustService_Enro
 	}
 
 	// OS-specific enrollment.
-	if initReq.DeviceData.OsType != devicepb.OSType_OS_TYPE_MACOS {
+	var cred *devicepb.DeviceCredential
+	var pub *ecdsa.PublicKey
+	switch initReq.DeviceData.OsType {
+	case devicepb.OSType_OS_TYPE_MACOS:
+		cred, pub, err = enrollMacOS(stream, initReq)
+		if err != nil {
```

Comment:
Maybe handle err after the switch? Same for the case below.

Reply:
0acf776324